### ISSUE TITLE
[Admin] [Back] develop git implementation

### ISSUE THREAD

**Issue by lucasmarcondes** [wwu-cs/Submitty#76]
#67 must be completed first.

We want to be able to access the Github specified folder and display the folder contents.

**Comment by dakriy** [comment]
Why is this admin? This seems like an instructor issue.

**Comment by dakriy** [comment]
Ok I think this is a duplicate of #72 since it seems to have depended on #67 which was completed and the backend is currently under review. So I'm gonna close this. If it shouldn't have been close please open a new more specific issue.
